core/templates/dev/head/components/rule_editor_directive.html

```diff
@@ -1,6 +1,9 @@
 <div style="position: relative;">
   <form class="form-inline protractor-test-rule-details" role="form" name="ruleEditForm.form">
-    <div ng-if="rule.type != 'FuzzyMatches'">
+    <div ng-if="rule.type != 'FuzzyMatches'" style="max-height: 350px;
+    overflow-y: scroll;
```

Comment:
Why not `overflow-y: auto`? As, in this case, won't  a disabled scroll bar be shown even when its not needed?